pubs/endecoder.py

```diff
@@ -115,11 +115,12 @@ def decode_bibdata(self, bibdata):
         try:
             try:
                 entries = bp.bparser.BibTexParser(
-                    bibdata, homogenize_fields=True,
-                    customization=customizations).get_entry_dict()
+                    bibdata, common_strings=True,
+                    customization=customizations,
+                    homogenize_fields=True).get_entry_dict()
             except TypeError:
```

Comment:
According to a5a653becd761bf56a7ac38e309c29a5ea3307a3, this was targeting older versions of bibtexparser. Since we now require a newer version, we should probably remove the exception catching altogether.

Reply:
Indeed. I am merging with this change included.